src/handlers/handleStatusPage.ts

```diff
@@ -79,6 +83,14 @@ function createEnvVarsInformationElement(env: IntegrationEnv): string {
       </span>
       `
     }
+
+    if (!isOpenClientResponseEnabled) {
```

Comment:
If a customer doesn't have the ability to use open client response, or just disabled it, status page will show a warning, saying that `OPEN_CLIENT_RESPONSE_ENABLED is not set`. This warning will show even if it is set to `false`. This doesn't sound right. 

Open Client Response is an optional feature, that's why we should do it the other way. We shouldn't care if it not enabled, and it would be _nice_ to show that it is enabled in the status page. In other words:
```
if (isOpenClientResponseEnabled) {
<span> <info> ${openClientResponseVarName} is set to true </info> </span>
}
```

Reply:
Also, showing integration info only for this flag doesn't make sense, and it's a warning that we will use default values because "you didn't provide any value".

Reply:
Refactored by this commit: https://github.com/fingerprintjs/fingerprint-pro-fastly-proxy-integration-example/pull/5/commits/f866bd3b72f76e9a6a71ae09bca8da82b8fc0513